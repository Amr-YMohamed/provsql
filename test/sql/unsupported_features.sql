\set ECHO none

SELECT * FROM (VALUES (1)) t, personal;

WITH Q(a) AS (SELECT 1) SELECT * FROM Q, personal;

SELECT COUNT(*) FROM personal;

SELECT * FROM personal WHERE city IN (SELECT city FROM personal);

SELECT DISTINCT ON (city) * FROM personal;

SELECT * FROM personal INTERSECT SELECT * FROM personal;
