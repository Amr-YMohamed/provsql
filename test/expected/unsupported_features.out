\set ECHO none
ERROR:  FROM clause unsupported by provsql
ERROR:  FROM clause unsupported by provsql
ERROR:  Aggregation not supported by provsql
ERROR:  Subqueries in WHERE clause not supported by provsql
ERROR:  DISTINCT ON not supported by provsql
ERROR:  Set operations other than UNION not supported by provsql
